### ISSUE TITLE
The "Try agent mode in VS Code" text is not visible after applying Text Spacing: A11y_VisualStudioCodeServices_Docs_RemoteDevlopmentTipsAndTricks_TextSpacing.

### ISSUE THREAD

**Issue by shindepra** [microsoft/vscode-docs#8432]
**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**

### GitHub Tags
#A11ySev2; #A11yTCS; #WCAG1.4.12; #Chrome; #DesktopWeb; #Win11; #Visual Studio Code Services; #BM_VisualStudioCodeServices_Web_May2025; #TextSpacing; #A11yWCAG2.1;

### Environment Details
Application Name: Visual Studio Code Services
URL: https://code.visualstudio.com/docs/remote/troubleshooting
Microsoft Windows 11 Enterprise (24H2) Build on 26100.4061
Chrome Version 136.0.7103.114 (Official Build) (64-bit)

### Repro Steps

1. Launch the application URL: https://code.visualstudio.com/docs/remote/troubleshooting
2. Tab till "Try agent mode in VS Code" link and press enter.
3. Apply Text spacing from the text spacing tool.
4. Observe that The "Try agent mode in VS Code" text is not visible after applying Text Spacing

### Actual Result
The "Try agent mode in VS Code" text is not visible after applying Text Spacing.

### Expected Result
The "Try agent mode in VS Code" text should be visible after applying Text Spacing.

### User Impact
User using text spacing will not be see the text present on the page if the "Try agent mode in VS Code" text is not visible after applying Text Spacing.

### Attachment
**After applying Text spacing**

![Image](https://github.com/user-attachments/assets/3eb93ddc-7ca1-43a4-98db-2510b4e14235)

**Before applying Text spacing**

![Image](https://github.com/user-attachments/assets/5f9541b2-f64b-478a-935f-b44117e14c17)

**Comment by ntrogh** [comment]
@hawkticehurst Can you look into this website issue?

**Comment by hawkticehurst** [comment]
Yep!

**Comment by hawkticehurst** [comment]
After reviewing I believe this is actually not a text spacing accessibility issue but rather an issue of the website's navbar becoming too large on tablet screen sizes and clipping the banner. There's an open issue for this problem in the VS Code website repo.

After applying the success criterion styles from [WCAG 1.4.12](https://www.w3.org/WAI/WCAG22/Understanding/text-spacing.html) this is what the banner looks like at different screen sizes.


Desktop screen size:

<img width="1728" alt="Image" src="https://github.com/user-attachments/assets/6d7e449b-101c-4949-8caf-195cdcd43eb1" />

Tablet screen size:

<img width="1728" alt="Image" src="https://github.com/user-attachments/assets/f13bca1e-c609-4098-b9d7-5820abf0db50" />

Mobile screen size:

<img width="1728" alt="Image" src="https://github.com/user-attachments/assets/08e1cf7f-9df4-4fef-9988-301aace89c4c" />

**Comment by hawkticehurst** [comment]
@shindepra if you could please retest this text spacing issue at different screen sizes to confirm my thinking, however, that would be appreciated :)

**Comment by hawkticehurst** [comment]
Ahh wait, looking back I realize what the issue is now. 

In the original screenshots I missed that text spacing is applied to the entire web page, so actually the issue is not with the announcement banner but rather with the navbar becoming too wide and overflowing/clipping the announcement banner.

It's still a result of the same issue with the navbar I just mentioned above, but I'll just go ahead and fix it now.

<img width="1728" alt="Image" src="https://github.com/user-attachments/assets/aeb1f59a-c3cb-4615-b572-0573a9e8328c" />

**Comment by ntrogh** [comment]
Thanks for looking into this @hawkticehurst !

**Comment by hawkticehurst** [comment]
This should be resolved with a PR that was just merged into the website repo. Waiting for the changes to be deployed before I close this out.

**Comment by hawkticehurst** [comment]
Changes have been deployed. @shindepra can you please verify that this resolves the issue?

**Comment by hawkticehurst** [comment]
Argh shoot, never mind. I just checked something I had forgotten to last week and this issue is still present. I'll work on fixing it today.

**Comment by alexr00** [comment]
@hawkticehurst moving to July.

**Comment by mrleemurray** [comment]
I had a quick look & an easy fix is to make the width of the nav container `1360px` instead of `1320px` to prevent wrapping

<img width="1707" height="879" alt="Image" src="https://github.com/user-attachments/assets/f6655043-653d-46ca-bd2f-6a9a2e39861f" />

**Comment by meganrogge** [comment]
Sounds good to me @mrleemurray thanks so much for investigating.

**Comment by hawkticehurst** [comment]
Amazing thanks @mrleemurray!

**Comment by Carloschapus** [comment]
As
